Notes for including SSL dependencies (#21262)
dotnet/docs: docs/core/deploying/trim-self-contained.md

</ItemGroup>
```

> ### Support for SSL Certificates

Reviewer: - Normal casing for titles.

```suggestion
### Support for SSL certificates
```

Author: Adding in the suggestion, thank you!

> Suppose one of the scenarios is to load SSL certificates within an ASP.NET Core app.  We want to ensure that when trimming we also prevent trimming assemblies that will help with loading SSL certificates.

Reviewer: - Remove double space
- I would be more aggressive with the wording. I feel like "Suppose" is too related to suggesting they may want to do something when the it's more like problem a exists here is a solution.

```suggestion
If your app loads SSL certificates, such as in an ASP.NET Core app, you'll want to ensure that when trimming you prevent trimming assemblies that will help with loading SSL certificates.
```

Author: Adding in the suggestion, thank you!
